plantcv/plantcv/classes.py

```diff
@@ -217,3 +216,28 @@ def __init__(self, array_data, max_wavelength, min_wavelength, max_value, min_va
 # spectral_array = Spectral_data(max_wavelength=1000.95, min_wavelength=379.027, d_type=numpy.float32,
 #                           wavelength_dict=dictionary, samples=1600, lines=1704, interleave='bil',
 #                           wavelength_units='nm', array_type="datacube", filename=fname, default_bands={159,253,520})
+
+
+class PSII_data:
+    """PSII data class"""
+
+    def __init__(self):
+        self.darkadapted = None
+        self.lightadapted = None
+        self.rgb = None
+        self.anthocyanin = None
+        self.chlorophyll = None
```

Comment:
@dschneiderch and @HaleySchuhl any objections if I add class attributes for `datapath` and `filename` (or similar). Then I would update `read_cropreporter` to return only `ps`. This would be consistent with where we want to move to with other input data

Reply:
No, I think thats better.
Wouldn't it be an instance attribute though?

Reply:
Cool, yep. I think I would just define them in the class like the others with a default value of `None` and then assign values in the instance.